src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -45,6 +44,13 @@ public static void GetDirectoryName(string path, string expected)
             Assert.Equal(expected, Path.GetDirectoryName(path));
         }
 
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework, ".Net Core returns null dotnet/corefx #19337")]
+        public static void GetDirectoryName_EmptyString_Throws()
+        {
+            Assert.Throws<ArgumentException>(() => Path.GetDirectoryName(string.Empty));
+        }
```

Comment:
It depends if it is an intended difference in behavior (a bug fixed that needs to port to desktop) I disable the test for desktop. 

If it is a breaking difference on behavior that is not intended I usually fix the test and the difference on behavior on the same PR sand have the discussion there as well.

Since this has to happen on coreclr I did it by opening issues and will solve it on that side. There is no rush, will fix the tests now and will wait for the fixes in coreclr to be in.